src/Form.js

```diff
@@ -97,7 +100,7 @@ export default class Form extends React.Component {
   modal = React.createRef();
 
   render() {
-    const { fullName, position } = this.state;
+    /* const { fullName, position } = this.state; */
```

Comment:
Why is this commented?

Reply:
Since I moved the modal part to a separate component this assignment is no longer needed.